server/src/models/results/populate.js

```diff
@@ -59,7 +59,7 @@ export default async function ({ models }) {
               indicator.stable_id === stable_id &&
               _.indexOf(time_period_ordered_doc_keys, indicator.doc) < doc_index
           )
-          .last()
+          .first()
```

Comment:
Ugh, wrote a nice long comment, but lost it.

Looking at this file, it was always relying on the order of the indicators within the csv. That was sloppy on my part, the sorting coming out of the pipeline must have reversed at some point (or this code was always wrong, doesn't mean it wasn't sloppy though).

If the sort ordering within the csv changes again, this will go back to being wrong. Shouldn't assume that either `first` or `last` is meaningful unless we've sorted the data ourselves first.

Code aside, the logic here is also wrong (unrelated to this bug, but worth correcting). If an indicator was dropped one year and revived later, this would still pick out the most recent historical copy. I may have figured that might be desired, but we call the fields "previous_year..." in the DB and label it as "Last year's ..." on the client, so this current logic is incorrect.

This is untested, but we probably want something like this
```
const previous_year_indicator_instance = _.find(
  indicator_records,
  (indicator) =>
    indicator.stable_id === stable_id &&
    indicator.doc === time_period_ordered_doc_keys[doc_index - 1]
)
```

`time_period_ordered_doc_keys[doc_index - 1]` is safe because this code's only reachable if `doc_index > 0`

Reply:
Ah actually this was the first solution I thought of, I guess it's safe